client/src/components/WebAuthn/Verify.scss

```diff
@@ -0,0 +1,52 @@
+$color-status-icon: rgba(0,113,196,1); // $color-blue from mfa module
```

Comment:
May as well centralise this since it's used in Register.scss too. Also note the "error" state variable and corresponding styles I mentioned above that were brought in from #33